airflow/providers/amazon/aws/hooks/batch_waiters.py

```diff
@@ -194,6 +201,9 @@ def wait_for_job(self, job_id: str, delay: int | float | None = None) -> None:
 
         :param delay:  A delay before polling for job status
 
+        :param get_batch_log_fetcher: A method that returns batch_log_fetcher of
+            type AwsTaskLogFetcher or None.
```

Comment:
This should explain what returning None from the getter means.

Reply:
I have updated it. Thanks!